src/exception/class-unverified-request-exception.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * File missing data in filter exception
+ *
+ * @package Eightshift_Forms\Exception
+ */
+
+declare( strict_types=1 );
+
+namespace Eightshift_Forms\Exception;
+
+use Eightshift_Libs\Exception\General_Exception;
+
+/**
+ * Unverified_Request_Exception class.
+ */
+class Unverified_Request_Exception extends \Exception implements General_Exception {
```

Comment:
Usually, I like to extend a specific exception. You can look at the list here

https://www.php.net/manual/en/class.exception.php

And pick the one you think suits the use case the most (I think `RuntimeException` would be good here).